### DISCUSSION TITLE
Unable to migrate server from docker arm to non-docker x86

### DISCUSSION THREAD

**Discussion by bitwarden48684** [dani-garcia/vaultwarden#2422] [Q&A] [answered]
I've got a server deployed in docker in arm, but I want to move that installation to an x86 machine. I've stopped the docker container, copied the files (database, attatchments, etc.) over to the new machine and launched the binary. Some migrations are run (even though the docker image was the latest version), and I'm able to reach the web vault, but my password is marked as incorrect. I've tried to migrate to docker in x86, and that works fine (doesn't run the migrations and I'm able to log in).

Can anyone help me out? I'd like to be able to restore backups to any server, without worrying about losing my passwords on restore.

Versions:
- ARM server:
  - Docker version: 20.10.14, build a224086349
  - Docker image: `vaultwarden/server:latest` (id 86445ffb6c1f, version )
  - Vaultwarden version: 1.24.0 (found by entering the docker container and running `/vaultwarden --version`)
- x86_64 server:
  - Vaultwarden version: 1.24.0
  - Rust version: 1.24.3
  - Installed on Arch Linux (see [build file for vaultwarden](https://github.com/archlinux/svntogit-community/blob/packages/vaultwarden/trunk/PKGBUILD))

**Comment by jjlin** [comment, marked as answer]
It sounds like you're putting your Vaultwarden data in the wrong place. According to the [build file](https://github.com/archlinux/svntogit-community/blob/8344c3f905b974a7c3d6e9f8e56f7086056cb0a4/trunk/PKGBUILD#L34), the data needs to be in `/var/lib/vaultwarden`.

**Reply by bitwarden48684** [reply]
It's true what they say about rubber ducks... While checking the config to respond to you, I've realized that the configuration file specifies that the database is located at `/var/lib/vaultwarden/data`, while the rest of the files are at `/var/lib/bitwarden`. Thank you for your time diagnosing my problem :)
